src/core/scene/a-scene.js

```diff
@@ -313,19 +313,29 @@ module.exports = registerElement('a-scene', {
         }
 
         this.addEventListener('loaded', function () {
-          if (this.renderStarted) { return; }
+          AEntity.prototype.play.call(this);  // .play() *before* render.
```

Comment:
Are you sure you want to start playing before rendering? What if there's components that rely on having an active camera?